src/ReactiveUI/ReactiveNotifyPropertyChangedMixin.cs

```diff
@@ -1,14 +1,9 @@
 using System;
 using System.Collections.Generic;
-using System.ComponentModel;
```

Comment:
I realise this is adding a bit more noise than ideal - is it worth doing a format of all files etc with rebracer config as a baseline?